gguf-py/gguf/constants.py

```diff
@@ -273,6 +275,21 @@ class MODEL_TENSOR(IntEnum):
         MODEL_TENSOR.FFN_DOWN,
         MODEL_TENSOR.FFN_UP,
     ],
+    MODEL_ARCH.STARCODER2: [
```

Comment:
(Probably not too important, but) in the other places/files, StarCoder2-specific stuff is added last, while here it's added next to the original StarCoder. Not sure which is better, but it's probably best to stay consistent.